lib/pdfjs-utils/domfacade.js

```diff
@@ -10,7 +10,7 @@ const utils = require("jsdom/lib/jsdom/living/generated/utils");
 const impl = utils.implSymbol;
 const HTMLElement = require("jsdom/lib/jsdom/living/generated/HTMLElement");
 const HTMLCanvasElement = require("jsdom/lib/jsdom/living/generated/HTMLCanvasElement");
-const Canvas  = require("jsdom/lib/jsdom/utils").Canvas;
+const { Canvas, Context2d } = require("jsdom/lib/jsdom/utils").Canvas;
```

Comment:
Canvas 2 also export `Image` and `ImageData` in a new way.
This is exported here https://github.com/ScientaNL/pdf-extractor/blob/master/lib/pdfjs-utils/domfacade.js#L199-L200

```suggestion
const { Canvas, Context2d, Image, ImageData } = require("jsdom/lib/jsdom/utils").Canvas;
```

Reply:
Can you change lines 199 and 200 too with after importing them from canvas:
```
exports.Image = Image;
exports.ImageData = ImageData;
```

Reply:
Oops, missed that. Ran the examples on the README to make sure.